os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -549,7 +560,8 @@ void board_initialize(void)
 	ipc_msg_loguart.msg_len = 1;
 	ipc_msg_loguart.rsvd = 0; /* for coverity init issue */
 	ipc_send_message(IPC_AP_TO_LP, IPC_A2L_DISLOGUART, &ipc_msg_loguart);
-
+	
+	up_check_iwdg();
```

Comment:
This function uses lldbg. It can be used :
1. before log uart initialized
2. at assert (when sw is not working properly)

Could you check this function is 1st case?

Reply:
This comes after log uart initialized, i will change it to dbg instead.